src/main/java/core/mate/academy/model/ExcavatorProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer {
```

Comment:
According to the checklist, you should not use raw types. Always parametrize interfaces and classes if it is possible. This code should be improved to: `public class ExcavatorProducer implements MachineProducer<Excavator> {`